Resources/Audio/DeltaV/Glimmer_Creatures/attributions.yml

```diff
@@ -0,0 +1,4 @@
+- files: [mite.ogg]
+  copyright: '"mite.ogg" by leonardo-dabepis'
+  license:
```

Comment:
put ` CC-BY-SA-3.0` or something after the :

Reply:
teehee

Reply:
I made it CC0 (public domain) because that's more based and it's a sound effect so I don't care as much